client/lib/main.dart

```diff
@@ -7,32 +7,33 @@ import 'package:firebase_analytics/firebase_analytics.dart';
 import 'package:firebase_analytics/observer.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:firebase_performance/firebase_performance.dart';
```

Comment:
I believe this is the only import that the PR added... but I can't see any corresponding code change that uses it. Is it needed?